### PR TITLE
Fix Demo 1 rendering, including figures

### PR DESCRIPTION
**Pull request by ashiklom**

### CODE REVIEW

**Review by robkooper — CHANGES_REQUESTED** [review]
can you change the image with the url and make it a 6480? we are trying to get rid of 32 bit machines (at least I am).

**Comment by mdietze** [comment]
@ashiklom when I pull this update it throws an error:
```
pandoc: Could not fetch ../documentation/tutorials/01_Demo_Basic_Run/startpecan.jpg
../documentation/tutorials/01_Demo_Basic_Run/startpecan.jpg: openBinaryFile: does not exist (No such file or directory)
Error: pandoc document conversion failed with error 67
Execution halted
```

File: documentation/tutorials/01_Demo_Basic_Run/Demo01.Rmd [hunk 1]
```diff
@@ -1,17 +1,16 @@
----
-title: 'PEcAn Hands-On Demo 01: Basic Run'
-output: 
-    html_document:
-        number_sections: yes
----
+# (PART) Tutorials {-}
```
Comment (mdietze) [inline documentation/tutorials/01_Demo_Basic_Run/Demo01.Rmd]: I really don't like the new title -- can we put it back?

File: documentation/tutorials/01_Demo_Basic_Run/Demo01.Rmd [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
 
-# Introduction {-}
+```{r setup}
+tutorial_dir <- '../documentation/tutorials/01_Demo_Basic_Run'
+```
 
-## Objective {-}
+# Introduction
```
Comment (mdietze) [inline documentation/tutorials/01_Demo_Basic_Run/Demo01.Rmd]: Drop "Introduction"

File: documentation/tutorials/01_Demo_Basic_Run/Demo01.Rmd [hunk 3]
```diff
@@ -32,8 +31,16 @@ In the following demo, **URL** is the web address of a PEcAn server and will ref
 2. **Click “Run Models”**
 3. **Click the ‘Next’ button** to move to the “Site Selection” page.
 
+```{r startpecan}
```
Comment (mdietze) [inline documentation/tutorials/01_Demo_Basic_Run/Demo01.Rmd]: Can we set echo to off on the code?

**Review by mdietze — CHANGES_REQUESTED** [review]

**Comment by ashiklom** [comment]
@mdietze Weird, it built fine on my local machine. I tried moving the tutorials into the `book_source` directory where all of the other documentation lives to see if that works. In theory, it shouldn't matter, but there may be a kink in our build process that makes it matter.

**Comment by mdietze** [comment]
@ashiklom I see that you've moved the tutorials out of the documentation folder. I'd really prefer if you'd put them back. A non-trivial part of the discussion around moving to Bookdown was the ability to link doc that are in other folders -- that's what I want us to be able to do.

**Comment by ashiklom** [comment]
Let's hold off on pulling this until after NACP.

**Pull request closed without merging**